dashboard/app/models/pd/workshop.rb

```diff
@@ -74,6 +74,19 @@ class Pd::Workshop < ActiveRecord::Base
     ]
   }
 
+  # Section types by course
+  SECTION_TYPE_MAP = {
+    COURSE_CSF => 'csf_workshop',
+    COURSE_CSP => 'csp_workshop',
+    COURSE_ECS => 'ecs_workshop',
+    COURSE_CS_IN_A => 'csina_workshop',
+    COURSE_CS_IN_S => 'csins_workshop',
+    COURSE_CSD => 'csd_workshop',
+    COURSE_COUNSELOR => 'counselor_workshop',
+    COURSE_ADMIN => 'admin_workshop'
+  }.freeze
+  SECTION_TYPES = SECTION_TYPE_MAP.values.freeze
```

Comment:
There's already a [workshop_type](https://github.com/code-dot-org/code-dot-org/blob/05974f17fd71727cdecdc6da5c04ef16abb6ab6d/dashboard/app/models/pd/workshop.rb#L90), either `Public`,`Private`, or `District`. Otherwise I'd agree.